templates/ecommerce/checkout.php

```diff
@@ -176,9 +171,8 @@
 						</div>
 						<?php endif; ?>
 						<!-- handle errors end -->
-
 						<button type="submit" id="tutor-checkout-pay-now-button" class="tutor-btn tutor-btn-primary tutor-btn-lg tutor-w-100 tutor-justify-center">
-							<?php esc_html_e( 'Pay Now', 'tutor' ); ?>
+							<?php echo esc_html( $is_zero_price ? __( 'Enroll Now', 'tutor' ) : __( 'Pay Now', 'tutor' ) ); ?>
```

Comment:
`$pay_now_btn_label       = __( 'Pay Now', 'tutor' );`
Please check
https://github.com/themeum/tutor/pull/1661/files#diff-a622ef087badd5c86bdfa938cd4d67ed1977b2f684218158db13770bdbf984df

We need to update this button label from trial plan checkout.